scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/nsop.yaml

```diff
@@ -0,0 +1,28 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'NSOP'
+
+NSOP-16_3.9x9.9mm_P1.27mm:
+  size_source: 'JEDEC MS-012AC, https://www.holtek.com/documents/10179/11842/HT42B534-xv110.pdf'
+  body_size_x: # from JEDEC
+    nominal: 3.9
+    tolerance: 0.1
+  body_size_y:
+    minimum: 9.8
+    maximum: 10
```

Comment:
Both body sizes are `BSC`. Their values are in squares, that means 'theoretically exact'.
I can't find the `0.1mm` tolerance in either datasheet, so please remove them.

The same applies to overall_size_x